### ISSUE TITLE
Plist not included with use_frameworks

### ISSUE THREAD

**Issue by rokgregoric** [InderKumarRathore/DeviceUtil#31]
When using `use_frameworks!`the plist is not included for some reason .. so the app stops at:
```
NSAssert(deviceList != nil, @"Please either use cocoapod for this library or include DeviceList.plist in your project");
```

**Comment by InderKumarRathore** [comment]
It's working, can you take the pod in new project and see. I have tested this on a new project and it was working. I'm using xcode 7.1.1

**Comment by rokgregoric** [comment]
Will try to create a test app that shows the problem.

**Comment by rokgregoric** [comment]
OK .. here it is: https://dl.dropboxusercontent.com/u/3547246/DeviceUtilPlist.zip super simple to recreate. Just created a Podfile with:
```
use_frameworks!
pod 'DeviceUtil'
``` 
and added:
```
import DeviceUtil
...
print("hardwareDescription: \(DeviceUtil.hardwareDescription())")
```
to the AppDelegate.swift .. and BOOM.

**Comment by InderKumarRathore** [comment]
I saw your pod file and it's missing target, 
Please use `pod init` command to create a pod file 
and the add  `pod 'DeviceUtil', '~> 1.3'` to your target
then `pod install`
here is the zip : https://www.dropbox.com/s/yb15g7x60fisam9/DUTest.zip?dl=1

**Comment by rokgregoric** [comment]
AFAIK the target is not required .. check out the sample here: https://guides.cocoapods.org/syntax/podfile.html

**Comment by InderKumarRathore** [comment]
Is it coming on simulator because I'm not getting this error
using pod 0.39.0 ad xcode 7.1.1

even if I use simple pod file like this


```
# Uncomment this line to define a global platform for your project
# platform :ios, '8.0'
# Uncomment this line if you're using Swift
use_frameworks!

pod 'DeviceUtil', '~> 1.3'


#target 'DUTest' do

#end
```

**Comment by InderKumarRathore** [comment]
Fixed in #33
